scripts/setup-torcharrow.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -efx -o pipefail
+# Some of the packages must be build with the same compiler flags
+# so that some low level types are the same size. Also, disable warnings.
+export CFLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt -w -std=c++17"  # Used by LZO.
```

Comment:
curious what's `-w`? -- see it in `setup-circleci.sh` but not in `setup-ubuntu.sh`: https://github.com/facebookincubator/velox/blob/1b0355a3fd161a6a3ee5a76009ad0fea733fad93/scripts/setup-ubuntu.sh#L21